### PR TITLE
Add a reviewers guide

### LINKED ISSUE #7: Add reviewer instructions
We need a set of instructions for reviewers of submitted inputs

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #7

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: docs/source/reviewersguide.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+A `protein.pdb` file should have been created for each system.
+
+You should check that the PDB has been prepped according to the instructions.
+To help you with this, you can use the `protein prep checking script <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/blob/reviewers-guide/industry_benchmarks/utils/maint/check_protein_prep.py>_`.
+
+This script will help you do the following:
+
+* Check that the number of atoms in each residue (excluding the termini)
+  does not change (i.e. protonation states have not changed).
+* Check that the positions of protein atoms have not change significantly.
```
Comment (hannahbaumann) [inline docs/source/reviewersguide.rst:80]:
```suggestion
* Check that the positions of protein atoms have not changed significantly.
```

**Review by hannahbaumann — COMMENTED** [review]
Looks good to me, just the small typo. One thing I noticed is that some people removed sodium or chloride ions from the PDB, some kept them. I think that if they are not close to the binding site it probably doesn't matter if they are kept or not since we add ions anyways, but maybe we could add a point about those here?

**Comment by IAlibay** [comment]
done @hannahbaumann - can you this another review / green tick if you approve?

**Review by hannahbaumann — APPROVED** [review]
Thanks, lgtm!

**Pull request merged**